Source/PropertyTools.Wpf/Controls/RadioButtonList/RadioButtonList.cs

```diff
@@ -301,22 +304,64 @@ private void UpdateContent()
                 var rb = new RadioButton
                 {
                     Content = content,
-                    Padding = this.ItemPadding
+                    Padding = this.ItemPadding,
                 };
 
+                var binding = CreateBindingFromOptionEnable(itemValue, enumType);
+                if (binding != null)
+                {
+                    rb.SetBinding(UIElement.IsEnabledProperty, binding);
+                }
+
                 var isCheckedBinding = new Binding(nameof(this.Value))
                 {
                     Converter = converter,
                     ConverterParameter = itemValue,
                     Source = this,
                     Mode = BindingMode.TwoWay
                 };
+
                 rb.SetBinding(ToggleButton.IsCheckedProperty, isCheckedBinding);
 
                 rb.SetBinding(MarginProperty, new Binding(nameof(this.ItemMargin)) { Source = this });
 
                 this.panel.Children.Add(rb);
             }
         }
+
+        /// <summary>
+        /// Creates a data binding for a WPF control based on the <see cref="OptionEnableByAttribute"/> applied to an enum value.
+        /// </summary>
+        /// <param name="itemValue">The value of the enum item to create the binding for.</param>
+        /// <param name="enumType">The type of the enum containing the item.</param>
+        /// <returns>
+        /// A data binding instance if the enum item has an <see cref="OptionEnableByAttribute"/>; otherwise, null.
+        /// </returns>
+        private Binding CreateBindingFromOptionEnable(object itemValue, Type enumType)

```

Comment:
I suggest to use the full name of the attribute type - "CreateBindingFromOptionEnableByAttribute"

Reply:
please also update the changelog!